pkg/trace/config/config.go

```diff
@@ -266,6 +266,8 @@ type EVPProxy struct {
 	DDURL string
 	// APIKey is the main API Key (defaults to the main API key).
 	APIKey string
+	// APIKey is the main API Key (defaults to the main API key).
```

Comment:
Should this comment be changed? At least it should say `ApplicationKey`

Reply:
```suggestion
	// ApplicationKey to be used for requests with the X-Datadog-NeedsAppKey set (defaults to the top-level Application Key).
```